release-notes/opensearch-dashboards-reports.release-notes-1.2.0.0.md

```diff
@@ -0,0 +1,40 @@
+### **Version 1.2.0.0 Release Notes**
```

Comment:
Is there an update in the 1.2 release notes to make the section titles in bold? Our past release notes don't have this style

Reply:
I followed the Version 1.1.0.0 Release Notes title style which i thought was in bold so went with it. I'll change it